Generator/AssociationTypeGenerator.php

```diff
@@ -21,6 +21,8 @@
  */
 class AssociationTypeGenerator implements GeneratorInterface
 {
+    const TYPE = 'association_types';
+
     const ASSOCIATION_TYPES_FILENAME = 'association_types.csv';
```

Comment:
why there are lot of things to rewrite ? just create a new writer no ? and if you use spout, it's very simple